serverless.yml

```diff
@@ -44,6 +44,13 @@ functions:
               status: false
               fromDateTime: false
               toDateTime: false
+  getTestResultsByTesterStaffId:
+    handler: src/functions/getTestResultsByTesterStaffId.getTestResultsByTesterStaffId
+    events:
```

Comment:
Are the events needed if we use the offline lambda invoke library?

Reply:
I am not sure. Let's keep it like this for the moment.